min.py

```diff
@@ -146,9 +178,28 @@ def get_min_scipy_version(min_python, min_numpy):
         MIN_PYTHON = get_min_python_version()
     MIN_NUMBA, MIN_NUMPY = get_min_numba_numpy_version(MIN_PYTHON)
     MIN_SCIPY = get_min_scipy_version(MIN_PYTHON, MIN_NUMPY)
+
     print(
         f"python: {MIN_PYTHON}\n"
         f"numba: {MIN_NUMBA}\n"
         f"numpy: {MIN_NUMPY}\n"
         f"scipy: {MIN_SCIPY}"
     )
+
+    pkgs = {
+        "numpy": "1.19",
+        "scipy": MIN_SCIPY,
+        "numba": MIN_NUMBA,
+        "python": MIN_PYTHON,
+        "python-version": MIN_PYTHON,
+    }
+
+    fnames = [
+        "pyproject.toml",
+        "requirements.txt",
+        "environment.yml",
+        ".github/workflows/github-actions.yml",
+        "README.rst",
+    ]
+
+    print(find_pkg_mismatches(pkgs, fnames))
```

Comment:
If we change the function signature above then we should put the `for` loop here:

```
for pkg_name, pkg_version in pkgs.items():
    print(find_pkg_mismatches(pkg_name, pkg_version, fnames))
```